### PR TITLE
Add SSO page to ~Teams~ Orgs

### PR DESCRIPTION
**Pull request by AlexTugarev**
This PR is a starting point for SSO integration into the dashboard.

## Description
 * Empty page
    Screenshot 2023-01-20 at 09 12 28

 * Configure new OIDC client
    Screenshot 2023-01-20 at 09 12 47

 * List existing OIDC clients 
    Screenshot 2023-01-20 at 09 12 53

 * Login button as a 🛹  
    Allows to try out, but breaks the iteration process❗ 
    Screenshot 2023-01-20 at 09 15 37

## Related Issue(s)

Part of #14278

## How to test
* navigate to https://at-oidc-config-ui.preview.gitpod-dev.com/
   * create a ~team~ org
   * go to Settings >  SSO
   * try to configure OIDC 
      * you can enter invalid entries, no validation included yet
      * to test the Login button, you'd need to setup OAuth2/OIDC in a playground project on https://console.cloud.google.com/

## Release Notes

```release-note
NONE
```

## Build Options:

- [ ] /werft with-github-actions
      Experimental feature to run the build with GitHub Actions (and not in Werft).
- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [x] /werft with-preview
- [ ] /werft with-large-vm
- [ ] /werft with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`

### CODE REVIEW

**Review by easyCZ — APPROVED** [review]

File: components/dashboard/src/teams/SSO.tsx [hunk 1]
```diff
@@ -0,0 +1,349 @@
...
+                    </div>
+                ) : null}
+            </div>
+
+            {clientConfigs.length === 0 && (
+                <div className="w-full flex h-80 mt-2 rounded-xl bg-gray-100 dark:bg-gray-900">
+                    <div className="m-auto text-center">
+                        <h3 className="self-center text-gray-500 dark:text-gray-400 mb-4">No OIDC Clients</h3>
+                        <div className="text-gray-500 mb-6">
+                            Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor
```
Comment (AlexTugarev) [inline components/dashboard/src/teams/SSO.tsx:154]: @jldec and @gtsiolis, please help with wording on this page.

Comment (jldec) [reply components/dashboard/src/teams/SSO.tsx:154]: #15995

**Pull request merged**